.github/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### Improvements
 
+* Implemented the `jaxpr_jvp` method to compute the adjoint jvp of a jaxpr using `lightning.qubit`.
```

Comment:
When starting to review I was a bit confused. In essence, there is nothing adjoint-specific in this PR, it's just intended to be used with adjoint exclusively, right? So I'd say maybe this is more accurate?
```suggestion
* Implemented the `jaxpr_jvp` method to compute the jvp of a jaxpr using `lightning.qubit`.
  This method currently only support the adjoint differentiation method.
```